pest/src/position.rs

```diff
@@ -17,16 +17,33 @@ use std::str;
 use span;
 
 /// A cursor position in a `&str` which provides useful methods to manually parse that string.
+#[derive(Clone)]
 pub struct Position<'i> {
     input: &'i [u8],
     pos: usize
 }
 
-pub unsafe fn new(input: &[u8], pos: usize) -> Position {
```

Comment:
Just want to note that removing this fn is a breaking change. 2.0 would have to keep it around in `#[deprecated]` format.

Reply:
Where else does it get used? I thought that because it is a private API removing it would be alright (as long as all references to it in the library itself were changed, which I think is the case).

Reply:
My bad I thought it was exported. (Can we please `pub(crate)` some things as we move to Edition2018)